qiskit_nature/operators/second_quantization/fermionic_op.py

```diff
@@ -645,6 +689,55 @@ def to_normal_order(self) -> FermionicOp:
             return ret
         return FermionicOp(("", 0), self.register_length, "sparse")
 
+    def normal_ordered(self) -> FermionicOp:
+        """Convert to the equivalent operator with normal order.
+
+        Returns a new operator (the original operator is not modified).
+        The returned operator is in sparse label mode.
```

Comment:
Is this really true? You are storing `self.display_format`, right?

Reply:
It's true. It looks like `self.display_format` is stored because it is changed in the function and changed back later. I actually copied this docstring and the code, verbatim, from `to_normal_order`. Do you want me to change the behavior?